jquery.canvaswrapper.js

```diff
@@ -166,7 +166,11 @@
                                 }
 
                                 if (positions.length == 0) {
-                                    delete styleCache[key];
+                                    if (styleCache[key].measured) {
+                                        styleCache[key].measured = false;
+                                    } else {
+                                        delete styleCache[key];
```

Comment:
So the key and its info are removed from the styleCache only when (all positions are inactive and then removed from the positions and the positions become an empty array) and (the info is measured).

I don't see how the info can't have width and height, but be measured. I think the measured flag is used to delay the deletion of the info. Something similar is done for the positions with a flag called 'active'. Maybe measured should be renamed to active to be consistent.